cmd/aro/operator.go

```diff
@@ -210,6 +211,11 @@ func operator(ctx context.Context, log *logrus.Entry) error {
 		if err = (muo.NewReconciler(arocli, kubernetescli, dh)).SetupWithManager(mgr); err != nil {
 			return fmt.Errorf("unable to create controller %s: %v", muo.ControllerName, err)
 		}
+		if err = (autosizednodes.NewReconciler(
+			log.WithField("controller", autosizednodes.ControllerName),
+			mgr)).SetupWithManager(mgr); err != nil {
+			return fmt.Errorf("unable to create controller AutoSizedNodes: %v", err)
```

Comment:
This is what we do with other reconcilers. Ok to update in a separate follow up PR so we don't have to wait for e2e again.

```suggestion
			return fmt.Errorf("unable to create controller %s: %v", autosizednodes.ControllerName, err)
```